### ISSUE TITLE
Release as conda package

### ISSUE THREAD

**Issue by ivanzvonkov** [nasaharvest/cropharvest#52]
Several packages in the cropharvest pip package (ie. rasterio, geopandas) require extensive steps to install on windows.

See:
- geopandas (<https://towardsdatascience.com/geopandas-installation-the-easy-way-for-windows-31a666b3610f>)
- rasterio (<https://iotespresso.com/installing-rasterio-in-windows/>)

Additionally, previously installed packages, the order these packages are installed manually, all impact whether the cropharvest pip package will run on windows.

In short, it makes sense to release the `cropharvest` package on conda which should take care of these installation issues.

**Comment by ivanzvonkov** [comment]
It turns out cropharvest can still be pip installed in windows as long as it's inside a conda environment and two packages (fiona, rasterio) are installed before. 

From README:

> Windows users must install cropharvest within a [conda](https://docs.conda.io/en/latest/miniconda.html) environment to ensure all dependencies are installed correctly:

> ```
> conda install 'fiona>=1.5' 'rasterio>=1.2.6'
> pip install cropharvest```

Since these lines address the underlying issue of running cropharvest on Windows, I'll close this issue. 

cc @sbaber1
